forgettable-frontend/src/pages/SignInPage/SignInPage.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classes from './SignInPage.module.css';
+import googleLogin from '../../assets/icons/google-login.svg';
+import logoBlack from '../../assets/logos/logo-black.svg';
+import {getAuth, signInWithPopup, GoogleAuthProvider} from 'firebase/auth';
+import {initializeApp} from 'firebase/app';
+
+
+export default function SignInPage() {
+  const signInHandler = ()=>{
+    // Configs to allow for firebase authentication
+    const firebaseConfig = {
+      apiKey: 'AIzaSyC1BtS4u1oIsbLSKA1TCzNK8f-PzfiXFOE',
```

Comment:
Don't commit the API key pls

Reply:
Should that be in a .env file?

Reply:
https://codewithandrea.com/tips/flutter-firebase-web-config-gitignore/
This is for flutter but same idea